src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,3 @@
+* {
```

Comment:
Consider using a more complete CSS reset or normalize library, such as 'normalize.css' or 'reset.css', to ensure consistent styling across different browsers.